File: api/domain/Appointment.cs
namespace Domain;
public class Appointment
{
    public int Id { get; set; }
    public string Title { get; set; }
    public DateTime StartTime { get; set; }
    public DateTime EndTime { get; set; }
    public string Service { get; set; }
    public int PatientID { get; set; }
    public int TherapistId { get; set; }
    public DateTime Date { get; set; }
    public string Location { get; set; }
    public string Status { get; set; }

    public TimeSpan GetDuration()
    {
        return EndTime.Subtract(StartTime);
    }

}

File: ui/src/app/common/models/Appointment.ts
export interface IAppointment {
    title: string;
    startTime: Date;
    duration: number;
    service: Service;
    patientId: string;
    createdByEId: string;
    notes: string;
    location: string;
    therapistEid: string;
    status: Status;
}

export enum Service {
    someservice,
    someotherservice
}

export enum Status {
    somestatus,
    someotherstatus
}